src/api/helpers/map-CIS-countries/map-CIS-country/map-short-term-cover-available/index.ts

```diff
@@ -0,0 +1,33 @@
+import { EXTERNAL_API_DEFINITIONS } from '../../../../constants';
+
+const {
+  CIS: { SHORT_TERM_COVER_AVAILABLE },
+} = EXTERNAL_API_DEFINITIONS;
+
+/**
+ * mapShortTermCoverAvailable
+ * Transform a countries 'short term cover available' string to a boolean
+ * @param {String} Risk category
+ * @returns {Boolean}
+ */
+export const mapShortTermCoverAvailable = (str: string): boolean => {
+  if (str === SHORT_TERM_COVER_AVAILABLE.YES) {
```

Comment:
could this be more efficient in a switch statement?

Reply:
Great shout, this was created ages ago, have updated to switch/case